### ISSUE TITLE
.kra support 

### ISSUE THREAD

**Issue by Daerpion** [AbyssalSoda/Canvas-File-Thumbnails#1]
Hello
Could you please add support for the .kra file format used by Krita? It shouldn't be hard since basically krita saves projects to a .zip archive and renames it to .kra. The zip archive even generates a preview.png inside of it which should be used accordingly.

Also is it possible to show a thumbnail of the file not only in graph but in a text editor?

Thanks in advance

**Comment by AbyssalSoda** [comment]
Yeah that doesn't seem to difficult to add, I'll definitely take a look at it. But as for the file in a "text editor", I assume you mean within the markdown file itself? As in a drag and drop a file to text to automatically create a hyperlink, a thumbnail of the file, or just a default emoji style thumbnail like a psd/clip logo?

**Comment by Daerpion** [comment]
> Yeah that doesn't seem to difficult to add, I'll definitely take a look at it. But as for the file in a "text editor", I assume you mean within the markdown file itself? As in a drag and drop a file to text to automatically create a hyperlink, a thumbnail of the file, or just a default emoji style thumbnail like a psd/clip logo?

I mean when you drag and drop png for example, it creates hyperlink in format ![[photo.png]] which shows thumbnail. Is the same possible with other file formats?

**Comment by AbyssalSoda** [comment]
Sorry for not getting back to you, I got caught up with other work and projects and never had much of chance to look at it. Yes, it should be possible to drag and drop .clip or .kra files and have a thumbnail display alongside or replace the file link, similar to how images are handled. I’ll add that to my to-do list. I also plan to revisit this plugin in late April and will include Krita support then.
